pkg/storage/internal/actions/create.go

```diff
@@ -60,8 +61,8 @@ func FromValidCreateActionArgs(args *wdk.ValidCreateActionArgs) CreateActionPara
 		LockTime:    args.LockTime,
 		Description: string(args.Description),
 		Labels:      args.Labels,
-		Outputs:     seq.PointersFromSlice(args.Outputs),
-		Inputs:      seq.PointersFromSlice(args.Inputs),
+		Outputs:     args.Outputs,
```

Comment:
wouldn't it be better to make it a pointer at this point?
```suggestion
		Outputs:     slices.Map(args.Outputs, to.Ptr),
```

Reply:
No, it's not needed.